### PR TITLE
Expose "address hash" with a public computed variable on PublicAddress

### PR DESCRIPTION
**Pull request by the-real-adammork**
Soundtrack of this PR: [Formally Unknown - Lotus Turbo [FAM07]](https://www.youtube.com/watch?v=Y4aLpV6SlEk)

### Motivation

Expose the "address hash" publicly. 

### In this PR
* computed var for the public address hash

### CODE REVIEW

File: Sources/Account/PublicAddress.swift [hunk 1]
```diff
@@ -58,6 +58,8 @@ public struct PublicAddress {
 
     public var fogReportUrlString: String? { fogInfo?.reportUrlString }
 
+    public var addressHash: String? { calculateAddressHash()?.hex }
```
Comment (remoun) [inline Sources/Account/PublicAddress.swift]: Let's name this to reflect that it's hex-encoded. We might want a deserialized or base64-encoded variant, to match the Android SDK. 
```suggestion
    public var addressHashHex: String? { calculateAddressHash()?.hex }
```

**Review by remoun — APPROVED** [review]
LGTM

File: Sources/Account/PublicAddress.swift (hunk 1 above)
Comment (the-real-adammork) [reply Sources/Account/PublicAddress.swift]: changed to a `Data` object then you can choose hex or base64 like below:

```swift
publicAddress.addressHash?.hexEncodedString()
publicAddress.addressHash?.base64EncodedString()
```

Comment (remoun) [reply Sources/Account/PublicAddress.swift]: Beautiful, thank you!

**Pull request merged**